File: WebAPI/WebAPI/Entities/Credit.cs
// Данный класс представляет собой кредит
public class Credit : Deposit
{
    private int PayedSum; // Сумма выплат клиента

    // Свойства
    public int payedSum
    {
        get { return PayedSum; }
        set { PayedSum = value; }
    }

    // Конструктор
    public Credit(int crID, int clID, string curr, int sum, int perc) : base(crID, clID, curr, sum, perc)
    {
        this.payedSum = 0;
    }
}

File: Front-end/src/entities/Credit.ts
class Credit{
    id: number;
    clientID: number;
    currency: string;
    sum: number;
    payedSum: number;
    percentage: number;

    constructor(ID: number, cID: number, curr: string, Sum: number, pSum: number, perc: number){
        this.id = ID;
        this.clientID = cID;
        this.currency = curr;
        this.sum = Sum;
        this.payedSum = pSum;
        this.percentage = perc;
    }
}